packages/core/nuxt-theme-module/theme/composables/useUiHelpers/index.ts

```diff
@@ -11,31 +11,36 @@ const useUiHelpers = () => {
 
   // eslint-disable-next-line
   const getCatLink = (category): string => {
-    console.warn('[VSF] please implement createThemeHelpers.getCatLink.');
+    console.warn('[VSF] please implement useUiHelpers.getCatLink.');
 
     return '/';
   };
 
   // eslint-disable-next-line
   const switchSorting = (sort) => {
-    console.warn('[VSF] please implement createThemeHelpers.switchSorting.');
+    console.warn('[VSF] please implement useUiHelpers.switchSorting.');
 
     return 'latest';
   };
 
   // eslint-disable-next-line
   const switchFilters = (filters) => {
-    console.warn('[VSF] please implement createThemeHelpers.switchFilters.');
+    console.warn('[VSF] please implement useUiHelpers.switchFilters.');
   };
 
   // eslint-disable-next-line
   const switchItemsPerPage = (itemsPerPage) => {
```

Comment:
isn't that more like a `change` instead of switch? talking purely about naming (and doesn't it apply to all fns here?

Reply:
I used word switch because it's related to the URL but can be `change` too